NOTICE

```diff
@@ -1,2 +1,7 @@
 Firecracker
-Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Just update it to 2021 directly?

Reply:
I don't think is correct to do that. Why not update to 2022?